cypress/e2e/title-level-request/check_cannot_change_tlr_modal.cy.js

```diff
@@ -0,0 +1,87 @@
+import { TestTypes, DevTeams, Permissions } from '../../support/dictionary';
+import { REQUEST_TYPES, FULFILMENT_PREFERENCES, REQUEST_LEVELS } from '../../support/constants';
+import UserEdit from '../../support/fragments/users/userEdit';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import Location from '../../support/fragments/settings/tenant/locations/newLocation';
+import Users from '../../support/fragments/users/users';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import SettingsMenu from '../../support/fragments/settingsMenu';
+import TitleLevelRequests from '../../support/fragments/settings/circulation/titleLevelRequests';
+import Requests from '../../support/fragments/requests/requests';
+
+describe('Title Level Request. Request notice. Title level request -Settings', () => {
+  let userData = {};
+  let requestId;
+  const testData = {
+    folioInstances: InventoryInstances.generateFolioInstances(),
+    servicePoint: ServicePoints.getDefaultServicePointWithPickUpLocation(),
+  };
+
+  before('Preconditions:', () => {
+    cy.getAdminToken();
+    ServicePoints.createViaApi(testData.servicePoint);
+    testData.defaultLocation = Location.getDefaultLocation(testData.servicePoint.id);
+    Location.createViaApi(testData.defaultLocation).then((location) => {
+      InventoryInstances.createFolioInstancesViaApi({
+        folioInstances: testData.folioInstances,
+        location,
+      });
+    });
+    cy.createTempUser([
+      Permissions.uiRequestsCreate.gui,
+      Permissions.uiRequestsView.gui,
+      Permissions.uiRequestsEdit.gui,
+      Permissions.requestsAll.gui,
+      Permissions.tlrEdit.gui,
```

Comment:
Permissions.tlrEdit.gui, - is not mentioned in test
Please remove

Reply:
I talk to functional tester - to exclude loginAsAdmin() and click Title Level Requset checkbox test user has to have this permission. Because if he hasn't - some other user (ex. admin) have to login and click on it and we don't have this step in test.